File: repo/ruby/v3_web.rb
# v3_web.rb
# Instalar: gem install sinatra humanize
# Ejecutar: ruby v3_web.rb
# Acceder:  http://localhost:4567/?n=10

require 'sinatra'
require 'humanize'

I18n.locale = :es

get '/' do
  numero = params['n'].to_i
  numero.humanize
end

File: repo/nodejs/v3_web.js
// v3_web.js
// Instalar: npm install
// Ejecutar: node v3_web.js
// Acceder:  http://localhost:3000/?n=10

const express        = require('express');
const { numeroALetras } = require('numero-a-letras');

const app  = express();
const PORT = 3000;

app.get('/', (req, res) => {
    const n = parseInt(req.query.n) || 0;
    res.send(numeroALetras(n));
});

app.listen(PORT, () => console.log(`Servidor en http://localhost:${PORT}`));
